File: src/main/java/common/Log.java
package common;

public class Log {

    public static void it(Object obj) {
        long time = System.currentTimeMillis() - CommonUtils.startTime;
        System.out.println(getThreadName() + " | " + time + " | " + "value = " + obj);
    }

    private static String getThreadName() {
        String threadName = Thread.currentThread().getName();
        if (threadName.length() > 30) {
            threadName = threadName.substring(0, 30) + "...";
        }
        return threadName;
    }

}

File: src/main/kotlin/common/Log_.kt
package common

class Log_ {

    companion object {

        fun it(obj: Any) {
            val time = System.currentTimeMillis() - CommonUtils_.startTime
            println(getThreadName() + " | " + time + " | " + "value = " + obj.toString())
        }

        private fun getThreadName(): String =
                Thread.currentThread().name.let {
                    when (it.length > 30) {
                        false -> it
                        true -> it.substring(0, 30) + "..."
                    }
                }

    }

}
